Sources/MuxPlayerSwift/FairPlay/FairPlaySessionManager.swift

```diff
@@ -37,7 +37,7 @@ protocol FairPlayStreamingSessionCredentialClient: AnyObject {
 
 // MARK: - PlaybackOptionsRegistry
 
-protocol PlaybackOptionsRegistry {
+protocol PlaybackOptionsRegistry: AnyObject {
```

Comment:
Is the `AnyObject` only necessary due to the tests?

Reply:
It's because the new `FairPlayStreamingSessionManager` must keep a weak reference to it, same as with the Credential Client

Reply:
Why does it need to be a weak reference?

Reply:
Right I think I see, it's needed to prevent a reference cycle because `ContentKeySessionDelegate` will keep two references to `DefaultFairPlayStreamingSessionManager` through both `credentialClient ` and `playbackOptionsRegistry`.

This makes me a bit hesitant because the potential reference cycle isn't an obvious one. Is there a way to combine  `credentialClient ` and `playbackOptionsRegistry` into one property on the delegate?

Reply:
Yup, I changed it around